godot_project/project_workspace/file_format/workspace_format_loader.gd

```diff
@@ -23,7 +23,7 @@ func _handles_type(typename: StringName) -> bool:
 
 
 func _load(path: String, _original_path: String, _use_sub_threads: bool, cache_mode: int) -> Workspace:
-	var tmp_path: String = path + ".tres"
+	var tmp_path: String = OS.get_cache_dir().path_join(path.get_file().get_basename() + ".tres")
```

Comment:
This will ensure the temp file is saved to a valid temp folder instead of trying to save it to `res://`, which is not possible because res:// is now contained inside of the pck file